plugins/flytekit-flyin/flytekitplugins/flyin/jupyter_lib/decorator.py

```diff
@@ -0,0 +1,60 @@
+import subprocess
+import sys
+from functools import wraps
+from typing import Callable, Optional
+
+from flytekit.loggers import logger
+from .constants import NO_ACTIVITY_TIMEOUT
+
+
+def jupyter(
+    _task_function: Optional[Callable] = None,
+    no_activity_timeout: Optional[int] = NO_ACTIVITY_TIMEOUT,
+    token: Optional[str] = "",
```

Comment:
can we not expose tokens for now? letting users specify token in python code is a bit strange. we can figure out the security afterwards

Reply:
ok, updated it.